staging/src/k8s.io/apiextensions-apiserver/pkg/apiserver/schema/pruning/algorithm_test.go

```diff
@@ -397,7 +416,7 @@ func TestPrune(t *testing.T) {
     }
   }
 }
-`, dontPruneMetaAtRoot: true, schema: &structuralschema.Structural{
+`, isResourceRoot: true, schema: &structuralschema.Structural{
```

Comment:
actually neither the old, nor the new name was good, the old one slightly more correct. What this say is that the prune algorithm will skip the TypeMeta and ObjectMeta fields and leave the pruning to our coercion algorithm.

Reply:
should it be `forcePruningAtRoot`?

Reply:
what it actually controls is whether the root of the schema is treated as a resource, regardless of the `XEmbeddedResource` setting